snapshots/output/basic-project/lib/other.dart

```diff
@@ -23,10 +23,13 @@
 //         ^^^^ reference local 0
       required this.value,
 //                  ^^^^^ reference scip-dart pub dart_test 1.0.0 lib/`other.dart`/Foo#value.
+//                  ^^^^^ definition scip-dart pub dart_test 1.0.0 lib/`other.dart`/Foo#`<constructor>`().(value)
```

Comment:
named parameters are indexed as both references and definitions, this is due to the case mentioned in #120, where they are goto'd from a declaration side of things